storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -222,3 +222,36 @@ include::modules/lvms-download-log-files-and-diagnostics.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data[About the must-gather tool]
+
+//Troubleshooting local persistent storage using LVM Storage
+
+include::modules/lvms-troubleshooting-persistent-storage.adoc[leveloffset=+1]
+
+include::modules/lvms-troubleshooting-investigating-a-pvc-stuck-in-the-pending-state.adoc[leveloffset=+2]
+
+include::modules/lvms-troubleshooting-recovering-from-missing-lvms-or-operator-components.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-lvmcluster_logical-volume-manager-storage[About the LVMCluster custom resource]
+
+* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#about-creating-lvmcluster-cr_logical-volume-manager-storage[Ways to create an LVMCluster custom resource]
+
+include::modules/lvms-troubleshooting-recovering-from-node-failure.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources-forced-cleanup-1"]
+.Additional resources
+
+* xref:../../../troubleshooting-local-persistent-storage-using-lvms.adoc#performing-a-forced-cleanup_logical-volume-manager-storage[Performing a forced clean-up]
```

Comment:
This link is not working for me in the Preview.

Reply:
Is this it?
```suggestion
* xref:../../..//storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.html#performing-a-forced-cleanup_logical-volume-manager-storage[Performing a forced clean-up]
```

Reply:
Fixed this link:

It should have been: 
"* xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#performing-a-forced-cleanup_logical-volume-manager-storage[Performing a forced clean-up]"